### PR TITLE
[roofit] fix doxygen warnings

### PR DESCRIPTION
**Pull request by ferdymercury**
# This Pull request:

## Changes or fixes:
Several warnings found by doxygen in roofit

## Checklist:

- [x] tested changes locally
- [x] updated the docs (if necessary)

### CODE REVIEW

File: roofit/roofitcore/src/RooDataHist.cxx [hunk 1]
```diff
@@ -1474,7 +1474,7 @@ void RooDataHist::set(std::size_t binNumber, double wgt, double wgtErr) {
 /// \deprecated Prefer set(std::size_t, double, double).
 /// \param[in] wgt New bin content.
 /// \param[in] wgtErr Optional error of the bin content.
-void RooDataHist::set(double wgt, double wgtErr) {
+void RooDataHist::set(Double_t wgt, Double_t wgtErr) {
```
Comment (hageboeck) [inline roofit/roofitcore/src/RooDataHist.cxx:1477]: I would remove that. RooFit is trying to go more in the direction of standard c++, but here you are going back.

**Review by hageboeck — CHANGES_REQUESTED** [review]
That's quite impressive work. There's one thing that should be changed, though.

For future PRs, I would recommend separating whitespace changes from documentation changes, from code changes. Like this, a reviewer can skip a commit such as `[RF][NCF] Fix training white space.` etc.

File: roofit/roofitcore/src/RooDataHist.cxx (hunk 1 above)
Comment (ferdymercury) [reply roofit/roofitcore/src/RooDataHist.cxx:1477]: I agree, but then the header file should be also with double instead of Double_t, no?

Comment (hageboeck) [reply roofit/roofitcore/src/RooDataHist.cxx:1477]: Yes! 🙂
But if you do this, please do it in a separate commit.

Comment (ferdymercury) [reply roofit/roofitcore/src/RooDataHist.cxx:1477]: I rebased to another branch https://github.com/root-project/root/pull/9657, closing this.

**Comment by ferdymercury** [comment]
> That's quite impressive work. There's one thing that should be changed, though.
> 
> For future PRs, I would recommend separating whitespace changes from documentation changes, from code changes. Like this, a reviewer can skip a commit such as `[RF][NCF] Fix trailing white space.` etc.

Sorry about the whitespaces, my text editor autostrips them when saving the file. :S

**Pull request closed without merging**